inst/Upgrades.qml

```diff
@@ -175,80 +175,150 @@ Upgrades
-                    case "excludePairwise":		return "pairwise";
-                    case "excludeListwise":		return "listwise";
-                    default:					return options["naAction"];
-                }
-            }
-        }
-    }
+	Upgrade
+	{
+		functionName:		"CorrelationBayesian"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
+
+		// MAIN SECTION - Additional options --------------
+		ChangeRename { from: "displayPairwise";						to: "pairwiseDisplay"						}
+		ChangeRename { from: "reportBayesFactors";					to: "bayesFactorReport"						}
+		ChangeRename { from: "flagSupported";						to: "supportCorrelationFlagged"				}
+		ChangeRename { from: "reportN";								to: "sampleSize"                            }
+		ChangeRename { from: "ciValue";								to: "ciLevel"                               }
+
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+				case "two.sided":				return "twoSided"	;
+				default:                        return options["alternative"];
+				}
+			}
+		}
+
+		// Main Plots
+		ChangeRename { from: "plotMatrix";								to: "matrixPlot"							}
+		ChangeRename { from: "plotMatrixDensities";						to: "matrixPlotDensity"						}
+		ChangeRename { from: "plotMatrixPosteriors";					to: "matrixPlotPosterior"					}
+
+		// Main section - Prior
+		ChangeRename { from: "kappa";									to: "priorWidth"							}
+
+		// PAIRS SECTION --------------
+		ChangeRename { from: "pairs";									to: "variablePairs" 						}
+
+
+		// Pairs Plots - Scatter plots
+		ChangeRename { from: "plotScatter";								to: "scatterPlot"							}
+		ChangeRename { from: "plotScatterAddInfo";						to: "scatterPlotAddInfo"					}
+
+		// Pairs Plots - Prior posterior plots
+		ChangeRename { from: "plotPriorPosterior";						to: "priorPosteriorPlot"					}
+		ChangeRename { from: "plotPriorPosteriorAddEstimationInfo";		to: "priorPosteriorPlotAddEstimationInfo"	}
+		ChangeRename { from: "plotPriorPosteriorAddTestingInfo";		to: "priorPosteriorPlotAddTestingInfo"		}
+
+		// Pairs Plots - Robustness plots
+		ChangeRename { from: "plotBfRobustness";						to: "bfRobustnessPlot"						}
+		ChangeRename { from: "plotBfRobustnessAddInfo";					to: "bfRobustnessPlotAddInfo"				}
 
+		// Pairs Plots - Sequential plots
+		ChangeRename { from: "plotBfSequential";						to: "bfSequentialPlot"						}
+		ChangeRename { from: "plotBfSequentialAddInfo";					to: "bfSequentialPlotAddInfo"				}
+
+
+		// OPTIONS SECTION --------------
+		ChangeRename { from: "missingValues";							to: "naAction"								}
+
+		ChangeJS
+		{
+			name:		"naAction"
+			jsFunction:	function(options)
+			{
+				switch(options["naAction"])
+				{
+				case "excludePairwise":		return "pairwise";
+				case "excludeListwise":		return "listwise";
+				default:					return options["naAction"];
+				}
+			}
+		}
+	}
+
+	Upgrade
+	{
+		functionName:		"RegressionLinear"
+		fromVersion:		"0.16.3"
+		toVersion:			"0.16.4"
```

Comment:
```suggestion
		fromVersion:		"0.16.4"
		toVersion:			"0.17.0"
```

Reply:
Can you fix this? Just need to make sure that the upgrader uses the correct versions

Reply:
Yes, I'm just waiting for the logistic regression PR to be merged first before I do this PR. There seems to be some conflicts.

Reply:
ok, makes sense :) I just saw you marked this as resolved, but we still need to do that. 

Anyway, the logistic regression is merged now so you can fix the conflicts

Reply:
done :)